pandas/core/dtypes/dtypes.py

```diff
@@ -152,13 +152,17 @@ class CategoricalDtype(PandasExtensionDtype, ExtensionDtype):
     ----------
     categories : sequence, optional
         Must be unique, and must not contain any nulls.
-        The categories are stored in an Index,
-        and if an index is provided the dtype of that index will be used.
+        The categories are stored in an Index.
     ordered : bool or None, default False
         Whether or not this categorical is treated as a ordered categorical.
         None can be used to maintain the ordered value of existing categoricals when
         used in operations that combine categoricals, e.g. astype, and will resolve to
         False if there is no existing ordered to maintain.
+    categories_dtype : dtype, optional
+        If given, will be the dtype of the categories.
```

Comment:
dtype is a better name

Reply:
I’d be ok to change, categories_dtype is very verbose.